scripts/relup-base-packages.sh

```diff
@@ -63,8 +63,20 @@ cd -P -- "$(dirname -- "${BASH_SOURCE[0]}")/.."
 mkdir -p _upgrade_base
 pushd _upgrade_base
 
+otp_vsn_for() {
+  local emqx_vsn="${1#[e|v]}"
+  local otp_vsns=($(../scripts/relup-base-vsns.escript otp-vsns-for "$emqx_vsn" ../data/relup_bases.eterm))
```

Comment:
we have `cd -P -- "$(dirname -- "${BASH_SOURCE[0]}")/.."` at the beginning of this script, why should we use `../scripts/...` here? `./scripts/...` should work ?

Reply:
Because we have just entered `_upgrade_base` in L. 64.